exercise_08/formatStringByWordsLength.js

```diff
@@ -16,3 +16,19 @@
  * CASE #2 * input: "This is short one."
  * output: "this is short one."
  */
+
+/**
+ * Check number of words contained in given string
+ * @param {string} sentence 
+ * @returns string
+ */
+const formatStringByWordsLength = function(sentence) {
+    const totalWords = sentence.split(' ').length;
+    
+    if (totalWords > 5) {
```

Comment:
Avoid magic numbers. 

```suggestion
    if (totalWords > THRESHOLD) {
```

- [ ] Extract number to constant